tox.ini

```diff
@@ -1,20 +0,0 @@
-[tox]
-envlist =
-    py27,py34,py35,py36
-
-[testenv]
-commands =
-    pip install coverage
-    # buildout's dev.py wants python to not have setuptools
-    pip uninstall -y zc.buildout setuptools pip
-    python dev.py
-    {toxinidir}/bin/test -1 -vvv -c
-    coverage combine
-    coverage report
-# since we're removing setuptools we can't possibly reuse the virtualenv
-recreate = true
-# if the user has ccache installed, PATH may contain /usr/lib/ccache that has a
-# gcc wrapper that fails to build anything when buildout's tests set HOME to a
-# non-existent directory under /tmp
-setenv =
-    PATH=/usr/bin:/bin
```

Comment:
I'm going to miss tox.

> tox is removed to avoid issues due to sharing the same directory between different virtual environments;

Which directory would that be?

Does it affect parallel tox runs only, or all tox runs?

Reply:
The way `tox` was configured, the repository root was shared between all `tox` environments.
Concretely, eggs for multiple Python versions would be installed in the `eggs` dir in your checkout.

I did not manage to find a way to solve that issue. Anyone with `tox` experience is welcome to revive it.

Reply:
Thanks for clarifying!